config.yml

```diff
@@ -63,23 +63,23 @@ params:
   bootstrap:            "https://getbootstrap.com"
 
   download:
-    source:             "https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/archive/v5.0.0-beta2.zip"
-    dist:               "https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/releases/download/v5.0.0-beta2/boosted-5.0.0-beta2-dist.zip"
-    dist_examples:      "https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/releases/download/v5.0.0-beta2/boosted-5.0.0-beta2-examples.zip"
+    source:             "https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/archive/v5.0.0-beta3.zip"
+    dist:               "https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/releases/download/v5.0.0-beta3/boosted-5.0.0-beta3-dist.zip"
+    dist_examples:      "https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/releases/download/v5.0.0-beta3/boosted-5.0.0-beta3-examples.zip"
 
   cdn:
     # See https://www.srihash.org for info on how to generate the hashes
-    css:                "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta2/dist/css/boosted.min.css"
+    css:                "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta3/dist/css/boosted.min.css"
     css_hash:           "sha384-W2o49ENLT+DBWaKVqWtw/XBJz9PJBOF02ro9QBp2CGs45l8vRt0mY81zZNjSLxnU"
-    css_rtl:            "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta2/dist/css/boosted.rtl.min.css"
+    css_rtl:            "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta3/dist/css/boosted.rtl.min.css"
     css_rtl_hash:       "sha384-hMAI+Pu7i4aPZaAfiGni7UwYKsGytZlffHihgERbYLtSYyjMldGGdEJQatII2Vvk"
-    helvetica:          "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta2/dist/css/orange-helvetica.min.css"
+    helvetica:          "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta3/dist/css/orange-helvetica.min.css"
     helvetica_hash:     "sha384-2W6Ddgejs+ZLUHWL9OWpfdx8K+DR+e1Muc2B220lXZmOgfxlJhYwD4YAC3AY5Kdx"
-    helvetica_rtl:      "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta2/dist/css/orange-helvetica.rtl.min.css"
+    helvetica_rtl:      "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta3/dist/css/orange-helvetica.rtl.min.css"
     helvetica_rtl_hash: "sha384-2N+Axfck8AYLrpdsZD0N1m7VB20fwJPh1SzLbOhtlxThb8GEzCaX4uJwMHLxCyxh"
-    js:                 "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta2/dist/js/boosted.min.js"
+    js:                 "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta3/dist/js/boosted.min.js"
     js_hash:            "sha384-VzJG8txEnR93rWKSsWA5q4qsd8HzVgTF9D8j96AfQRwWr9a5M6IFgeJnd4Sbpg4P"
-    js_bundle:          "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta2/dist/js/boosted.bundle.min.js"
+    js_bundle:          "https://cdn.jsdelivr.net/npm/boosted@5.0.0-beta3/dist/js/boosted.bundle.min.js"
     js_bundle_hash:     "sha384-Vc9TemN3pAGe1Yu1xsF6n3b8t65sxppt3SP/znqPkSmcNdTD9kKa3XqdlegP0jqP"
     popper:             "https://cdn.jsdelivr.net/npm/@popperjs/core@2.8.2/dist/umd/popper.min.js"
```

Comment:
Strange if this is a merge from bootstrap beta3 tag, poper need to be updated https://github.com/twbs/bootstrap/blob/v5.0.0-beta3/config.yml#L78 and his hash too

Reply:
Modified manually